Dockerfile

```diff
@@ -29,13 +30,16 @@ COPY function-template ${FUNCTION_TEMPLATE}/
 
 COPY validator /root/validator/
 
-ENV WORKDIR=/root/function PORT=8080
+ENV WORKDIR=/root/function PORT=8080 SERVERS=$servers
 EXPOSE ${PORT}
 WORKDIR ${WORKDIR}
 
 COPY ./index.ps1 /root
 
+RUN curl -L https://github.com/dispatchframework/funky/releases/download/0.1.1/funky0.1.1.linux-amd64.tgz -o funky0.1.1.linux-amd64.tgz
+RUN tar -xzf funky0.1.1.linux-amd64.tgz
```

Comment:
Should probably store the funky version in a variable so its easy to change in the future.

Reply:
good call